tools/scripts/build_base_image.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+ip=${1}
+port=${2:8585}
+
+# create http server
+nohup python3 -m http.server --directory /data2/shared/nvidia $port > tmp.log 2>&1
+
+export TENSORRT_URL=http://$ip:$port/TensorRT-8.2.3.0.Linux.x86_64-gnu.cuda-11.4.cudnn8.2.tar.gz
+export TENSORRT_VERSION=8.2.3.0
+export CUDA_INT=113
+export TAG=ubuntu20.04-cuda11.3
```

Comment:
If we upgrade the 3rdparty libs, will the tag be updated?

Reply:
It seems the tag should be update, not sure how to update the tag. What do you think of this way?
```
tag=ubuntu20.04-cuda11.3 means the latest
tag=ubuntu20.04-cuda11.3-2023xxxxx means the old version
```
each time we build image, we give tag the image twice.

Reply:
It's an option. But how can we know what the image includes by its tag? should we maintain an overview like other popular docker images, for example https://hub.docker.com/_/ubuntu

Reply:
Yes, we could add comment for each update on the overview page.